### PR TITLE
Removed alpha from ForceShiftedLJ and LJ

### LINKED ISSUE #800: Remove alpha from DPDLJ and ForceShiftedLJ
## Description

Remove the alpha parameter from DPDLJ and ForceShiftedLJ.

## Motivation and context

We no longer use the alpha parameter. Users should instead set r_cut per type pairs.

### PR DESCRIPTION
**Pull request by pepak13**
## Description
Removed the alpha parameter from the ForceShiftedLJ and LJ 

Added the struct param_type for ForceShiftedLJ as it wasn't there before. 

closes #800 

## Motivation and context

Alpha no longer a feature in Hoomd3.0

## How has this been tested?

Ran test sims of simple spheres using changed pair potentials

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/master/ContributorAgreement.md).
- [x] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

File: hoomd/md/EvaluatorPairForceShiftedLJ.h [hunk 1]
```diff
@@ -48,7 +48,52 @@ class EvaluatorPairForceShiftedLJ
...
+            v["epsilon"] = lj2 / (sigma6 * 4.);
+            return v;
+            }
+#endif
+        }
+#ifdef SINGLE_PRECISION
+    __attribute__((aligned(8)));
+#else
+    __attribute__((aligned(16)));
+#endif
```
Comment (joaander) [inline hoomd/md/EvaluatorPairForceShiftedLJ.h:96]:
```suggestion
typedef EvaluatorPairLJ::param_type param_type;
```
We can revert this change. This line of code borrows the identical param_type struct from `EvaluatorPairLJ` without duplicating code.

**Review by joaander — CHANGES_REQUESTED** [review]
Thanks!

Sorry for the ill-defined issue, it must have been out of date with respect to other changes in the code. I found and removed a couple more uses of alpha in comments to avoid confusion. I have one change request.

**Comment by joaander** [comment]
@mariaward10 do you have any questions about reviewing this pull request?

**Review by mariaward10 — APPROVED** [review]
I went over the changes and they look good, seem ready to go.

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**